lib/HeadersGeneration/ModelToHeaderPipe.cpp

```diff
@@ -76,3 +77,40 @@ static pipeline::RegisterDefaultConstructibleContainer<ModelHeaderFileContainer>
 } // namespace revng::pipes
 
 static pipeline::RegisterPipe<revng::pipes::ModelToHeader> Y;
+
+namespace revng::pypeline::pipes {
+
+ObjectDependencies ModelToHeader::run(const Model &TheModel,
+                                      const Request &Incoming,
+                                      const Request &Outgoing,
+                                      llvm::StringRef DynamicConfig,
+                                      RootBuffer &Buffer) {
+  if (Outgoing[0].empty())
+    return {};
+
+  revng_assert(Outgoing[0].size() == 1);
+  ObjectID Target = *Outgoing[0][0];
+  const model::Binary &Binary = *TheModel.get().get();
+  revng::Tracking::clearAndResume(Binary);
```

Comment:
We can't expect devs to do tracking stuff by hand.

Let's discuss how to do this in person.